src/database/memory.rs

```diff
@@ -493,7 +493,10 @@ macro_rules! populate_test_db {
         let input = if $is_coinbase {
             vec![$crate::bitcoin::TxIn::default()]
         } else {
-            vec![]
+            vec![$crate::bitcoin::TxIn {
```

Comment:
This is not really well-formatted.

Also, regarding the commit name: "make changes" is not really informative...